controllers/networkplugins_controller.go

```diff
@@ -48,7 +48,8 @@ import (
 )
 
 const (
-	DefaultNamespace        = "default"
+	DefaultNamespace        = "luigi-system"
+	KubemacpoolNamespace    = "dhcp-controller-system"
```

Comment:
Shouldn't both be defaulting to same NS here? We can just use same NS that the user selects for dhcp plugin

Reply:
dhcpController's namespace is fixed to dhcp-controller-system and is not specified in the network plugins yaml. Since kubemacpool's NS can be configured, the default has to be explicitly set to dhcp-controller-system